resources/views/home.blade.php

```diff
@@ -262,60 +260,13 @@ class="tech-icon h-full w-12 m-auto grayscale hover:grayscale-0 object-contain t
-                    linkedIn="{{ $testimonials[0]['linkedIn'] }}"
-                />
+                <x-reference @if(count($references)!=0) :reference="$references[0] @endif "/>
             </div>
-            <div class="hidden sm:block main-gallery js-flickity" data-flickity-options='{ "wrapAround": true }'>
-                <div class="gallery-cell gap-4 py-4 2xl:gap-12">
-                    <x-testimonial
-                        photoUrl="{{ $testimonials[0]['photoUrl'] }}"
-                        name="{{ $testimonials[0]['name'] }}"
-                        occupation="{{ $testimonials[0]['occupation'] }}"
-                        description="{{ $testimonials[0]['description'] }}"
-                        linkedIn="{{ $testimonials[0]['linkedIn'] }}"
-                    />
-                </div>
-                <div class="gallery-cell gap-6 py-4 2xl:gap-12">
-                    <x-testimonial
-                        photoUrl="{{ $testimonials[1]['photoUrl'] }}"
-                        name="{{ $testimonials[1]['name'] }}"
-                        occupation="{{ $testimonials[1]['occupation'] }}"
-                        description="{{ $testimonials[1]['description'] }}"
-                        linkedIn="{{ $testimonials[1]['linkedIn'] }}"
-                    />
-                </div>
-                <div class="gallery-cell gap-6 py-4 2xl:gap-12">
-                    <x-testimonial
-                        photoUrl="{{ $testimonials[2]['photoUrl'] }}"
-                        name="{{ $testimonials[2]['name'] }}"
-                        occupation="{{ $testimonials[2]['occupation'] }}"
-                        description="{{ $testimonials[2]['description'] }}"
-                        linkedIn="{{ $testimonials[2]['linkedIn'] }}"
-                    />
-                </div>
-                <div class="gallery-cell gap-6 py-4 2xl:gap-12">
-                    <x-testimonial
-                        photoUrl="{{ $testimonials[0]['photoUrl'] }}"
-                        name="{{ $testimonials[0]['name'] }}"
-                        occupation="{{ $testimonials[0]['occupation'] }}"
-                        description="{{ $testimonials[0]['description'] }}"
-                        linkedIn="{{ $testimonials[0]['linkedIn'] }}"
-                    />
-                </div>
-                <div class="gallery-cell gap-6 py-4 2xl:gap-12">
-                    <x-testimonial
-                        photoUrl="{{ $testimonials[2]['photoUrl'] }}"
-                        name="{{ $testimonials[2]['name'] }}"
-                        occupation="{{ $testimonials[2]['occupation'] }}"
-                        description="{{ $testimonials[2]['description'] }}"
-                        linkedIn="{{ $testimonials[2]['linkedIn'] }}"
-                    />
-                </div>
+            <div @class([ 'sm:block main-gallery js-flickity' => count($references)>3, 'sm:flex justify-center' => count($references)<=3])
```

Comment:
```suggestion
            <div @class([ 'sm:block main-gallery js-flickity' => $references->->count() > 3, 'sm:flex justify-center' =>  $references->->count() <= 3])
```

Reply:
For me, better should be moved that into variable in controller like `$referencesCount` and use them here